jetty9-compat-base/src/main/java/com/google/apphosting/vmruntime/jetty9/VmRuntimeWebAppContext.java

```diff
@@ -359,25 +362,6 @@ public RequestContext getRequestContext(Request baseRequest) {
       this.requestSpecificEnvironment = VmApiProxyEnvironment.createFromHeaders(System.getenv(),
           metadataCache, this, VmRuntimeUtils.getApiServerAddress(), wallclockTimer,
           VmRuntimeUtils.ONE_DAY_IN_MILLIS, defaultEnvironment);
```

Comment:
in the default mode, if there is a request ticket, it will be used by all code run from within the request scope, including async callbacks.

in the use_global mode, the global ticket is always used.